ironfish/src/fileStores/config.ts

```diff
@@ -23,6 +23,7 @@ export type ConfigOptions = {
   enableRpc: boolean
   enableRpcIpc: boolean
   enableRpcTcp: boolean
+  enableSyncer: boolean
```

Comment:
this does more than enable or disable the syncer, but I guess it's okay for now to call it this. It just requires a caveat, that it really disables all syncing.

Reply:
I'll rename this to `enableSyncing`